cas-server-webapp/src/main/webapp/css/cas.css

```diff
@@ -525,3 +528,35 @@ footer a:link, footer a:visited {
     white-space: nowrap;
     text-align: center;
 }
+
+.hr-text {
```

Comment:
The style for `<hr>` with text in the middle. Here is the [example](https://codepen.io/scottzirkel/pen/yNxNME) I followed.